templates/profile.php

```diff
@@ -114,6 +114,28 @@ function pmpromd_wp_title($title, $sep)
 			return $title;
 		}
 		add_filter("wp_title", "pmpromd_wp_title", 10, 2);
+
+		/**
+		 * We're working with the menu now so remove the filters
+		 */
+		function pmpromd_remove_filters_menu_title( $nav_menu, $args ) {	

```

Comment:
```suggestion
		/**
		 * We're working with the menu now so remove the filters.
		 *
		 * @since TBD
		 *
		 * @param string|null $output Nav menu output to short-circuit with. Default null.
		 */
		function pmpromd_remove_filters_menu_title( $nav_menu ) {	

```